tests/cases/dom/manipulate/toggleClass.js

```diff
@@ -33,6 +33,16 @@ QUnit.test(
 
         toggleClass(element, "foo", false);
         assert.strictEqual(element.classList.length, 0, "class wasn't removed with toggleClass");
```

Comment:
This comment is wrong?

```suggestion
        assert.strictEqual(element.classList.length, 0, "class wasn removed with toggleClass");
```

Reply:
Isn't the third parameter, the message, the one that is displayed in case the assertion fails? In that case, the message seems correct to me as removing the CSS class using `toggleClass` has failed.

Reply:
No, it's exactly the other way round. 😉 
This message describes what this test does / what is expected.

It is correct that it is (normally) displayed in case of an error with "I expected ABC to happen, but it didn't", but if you are running `ava -v` (which you should tbh) then you can see all the messages.

Reply:
(or in QUnit, what we are using here)